File: typescript/21-capitalize-first-letter.ts
// Capitalizar primera letra — Fácil · Strings
// Capitaliza la primera letra sin modificar el resto del string.
// Fuente: https://coding-challenges.dev/problems/capitalizar-primer-letra

function capitalizeFirstLetter(text: string): string {
  return text.charAt(0).toUpperCase() + text.slice(1);
}

// No modificar: necesario para evaluar el resultado.
export { capitalizeFirstLetter };

File: python/21_capitalize_first_letter.py
"""Capitalizar primera letra — Fácil · Strings

Devuelve el string con la primera letra en mayúscula y el resto sin modificar.
Un string vacío devuelve otro string vacío.

Fuente: https://coding-challenges.dev/problems/python-capitalizar-primer-letra
"""


def capitalize_first_letter(text: str) -> str:
    return text[:1].upper() + text[1:]
